config/projects.yml

```diff
@@ -336,7 +338,9 @@ page-bundle:
         sonata_admin: ['3']
         sonata_block: ['3']
 
-sandbox: ~
+sandbox:
+  excluded_files:
```

Comment:
@Soullivaneuh is it possible to exclude files per branch? or not yet?

because comment-bundle `2.x` is using the old folder structure

Reply:
ah forget it, 2.x isn't managed via dev-kit anymore, cause its legacy 👍